sklearn/metrics/tests/test_common.py

```diff
@@ -910,6 +910,9 @@ def test_thresholded_invariance_string_vs_numbers_labels(name):
 )
 @pytest.mark.parametrize("y_true, y_score", invalids_nan_inf)
 def test_regression_thresholded_inf_nan_input(metric, y_true, y_score):
+    if metric == coverage_error:
+        y_true = [y_true]
+        y_score = [y_score]
```

Comment:
I used this condition to make tests pass, but it might add cognitive complexity. Just wondering any other approach.

Reply:
Thanks for the suggestions!